server/routes/terraform/terraform_manager.py

```
@@ -118,7 +118,7 @@ def _ensure_terraform_installed(self) -> bool:
                 shutil.copy(terraform_binary, "/usr/local/bin/terraform")
                 
                 # Make Terraform executable
-                os.chmod("/usr/local/bin/terraform", 0o755)
+                os.chmod("/usr/local/bin/terraform", 0o750)
```

Ruff (0.15.10)

[error] `os.chmod` setting a permissive mask `0o750` on file or directory

(S103)